src/js/with-react.js

```diff
@@ -162,6 +186,14 @@ const withReact = (WebComponent, { pure = true, passive = false, eventNamespace
     render() {
       // eslint-disable-next-line react/prop-types
       const { props: { children, ...props }, handleRef } = this;
+
+      // bail-out if we are in a test-environment
+      // @todo: we have to differentiate between our test-environment and consumer test-environment
```

Comment:
This is not ideal, because if we want to test the pattern lib ourselves it won't be possible...